app/tut/ga/cad-e-riomhchlaru.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Cad é Ríomhchlárú?
+prev: reamhra
+prev-text: Réamhrá Teagaisc
+---
+
+# Is é Ríomhchlárú ná...
+
+Sula dtosóimid ag foghlaim conas Setanta a scríobh agus a úsáid, ba chóir dúinn an ceist seo a chur:
```

Comment:
an cheist seo a chur

Reply:
D'athraigh mé an líne seo sa leagan is-déanaí